source/documentation/apps/manage-app-settings-from-cpanel.md

```diff
@@ -0,0 +1,108 @@
+<div style="height:0px;font-size:0px;">&nbsp;</div>
+
+If you choose to use `email` (`passwordless` flow),  then you can manage the app's customers for each deployment environment through clicking `Manage customers` button on [app-list page](https://controlpanel.services.analytical-platform.service.justice.gov.uk/webapp-data/). 
+
+Right now we only provide customer management for `email` login option. If you choose other options like `nomis` or `github` etc, then it means your app will be open to any users who have nomis credential or who has a Github account and has joined the `moj-analytical-services` Github org.  Further user management and control is required under app-level if the default scope of users is wider than the target audience of the app.
+
+### IP whitelist
+
+You can configure whether your app needs extra protection from internet environment by setting the allowed IP_RANGES (the list VPN managed in MoJ). You can set up this option even if your app is public facing (`AUTHENTICATION_REQUIRED` is `False`)
+
+<div style="height:0px;font-size:0px;">&nbsp;</div>
+| Command | Format | Description |
+|------------------|---------------------------------------------|
+| `IP_RANGES`      | github secret | The list of MoJ VPNS being allowed to this app, editable|
+<div style="height:0px;font-size:0px;">&nbsp;</div>
+
+### Self-defined secrets or environment vars
+
+If the app has its own settings and the value of the setting depends on which deployment environment the app is running in, you can create secret (sensitive value e.g., credential or api-key) or environment variable (non-sensitive value) through the app-detail page. 
+
+### Other Github secrets and environment vars
+
+Other secrets and vars which are not mentioned in the section, if they are not the ones you defined, e.g. having `ECR` or `KUBE` or `AWS` as part of the name,  then they are created during the process of initialising the infrastructure resources on CP and are required by the deployment workflows too.  They are maintained or updated usually through the terraforms of CP.
+
+## Want to make change to `AUTHENTICATION_REQUIRED` flag?
+
+If the flag was off (`False` for `AUTHENTICATION_REQUIRED`) and you switch it on,  then an auth0-client is required to be created by clicking `Create auth0 client` on app-detail page on Control panel. If the client is missing,  a red warning flag will be displayed on the page to remind you
+
+If the flag was on (`True` for `AUTHENTICATION_REQUIRED`) and you swtich it off,  then existing auth0-client becomes redundant and please do remove it by clicking the `Remove the auth0 client` to save resource on auth0 platform. 
+
+## Can I make changes(add/remove/update) the secrets/vars on github repo directly?
+
+As the deployment settings are stored in the Github repo and not in Control panel,  any changes made on the Github repo will be reflected back in Control panel.
```

Comment:
```suggestion
As the deployment settings are stored in the Github repo and not in Control panel,  any changes made on the Github repo will not be reflected back in Control panel.
```

Reply:
@julialawrence,  it will be reflected back to control panel as control panel  doesn't store any values of those settings, it just read/update on github repo directly,  so if someone has the admin of repo and he/she changes the value directly on repo,  then control panel will read new values. 

 maybe I shouldn't use word `reflect` ?